staging/src/k8s.io/kubectl/pkg/cmd/portforward/portforward.go

```diff
@@ -276,19 +276,20 @@ func checkUDPPortInService(ports []string, svc *corev1.Service) error {
 // checkUDPPortInPod returns an error if remote port in Pod is a UDP port
 // TODO: remove this check after #47862 is solved
 func checkUDPPortInPod(ports []string, pod *corev1.Pod) error {
```

Comment:
Nice catch! Thanks for you fix! Would apply the same fix to `checkUDPPortInService`?

/hold

Reply:
Right @knight42, I did not see that one, I'll update the PR. Thanks